### ISSUE TITLE
Automatically run examples in docs?

### ISSUE THREAD

**Issue by cbrnr** [cbrnr/sleepecg#117]
I don't know if this is even possible, but it would be nice to run the examples in the docs automatically and include their output (e.g. figures, text output, ...) in the docs. Currently, this has to be done manually.

**Comment by hofaflo** [comment]
This would be nice, but e.g. the benchmarks take a really long time and don't change that often, so I think this would be overkill for the current state of the project.

**Comment by cbrnr** [comment]
I agree. Sure we could exclude the benchmarks, but even then I don't think it is worth the effort right now.

**Comment by cbrnr** [comment]
I'm reopening this, because I think it is useful to run the smaller examples and include their output. It looks like https://pawamoy.github.io/markdown-exec/ can be used for that purpose.
